ctree.h

```diff
@@ -40,6 +40,10 @@
 #include "async-thread.h"
 #include "version_compat.h"
 
+#if BTRFS_RHEL_VERSION_CODE >= BTRFS_RHEL_KERNEL_VERSION(3,10,0,693,0,0)
+#include <linux/btrfs_tree.h>
```

Comment:
If it merges cleanly, we should just merge btrfs_tree.h into ctree.h in px-btrfs.